packages/assets-api/README.md

```diff
@@ -0,0 +1,74 @@
+# @talend/assets-api
+
+Assets are files from a npm package which can be needeed at some point by a web application. It can be a stylesheet, a SVG icon, a js script, a json file, etc...
```

Comment:
```suggestion
Assets are files from a npm package which can be needed at some point by a web application. It can be a stylesheet, a SVG icon, a js script, a json file, etc...
```